build.sh

```diff
@@ -156,8 +143,23 @@ while [[ $# > 0 ]]; do
       shift
       ;;
     -branding)
-      branding="$2"
-      SetBranding "$branding"
+      if [ -z ${2+x} ]; then
+        echo "No branding type supplied. See help (--help) for supported values." 1>&2
+        exit 1
+      fi
+      passedBrandingType="$(echo "$2" | tr "[:upper:]" "[:lower:]")"
+      case "$passedBrandingType" in
+        repodefault|unstable|preview|release)
+          ;;
+        *)
+          echo "Unsupported branding type '$2'."
+          echo "The allowed values are repodefault, unstable, preview or release."
+          echo ""
+          echo "IMPORTANT: If you previously passed in the '-branding rtm' argument for .NET 10, remove it entirely. The default is now set correctly (VMR branding defaults for release branch builds or otherwise repo branding defaults)."
```

Comment:
@MichaelSimons this is now the 1xx PR which would introduce the build breaking change for anyone who previously passed `--branding rtm` in.

Based on what @omajid shared [here](https://github.com/dotnet/dotnet/pull/3574#issuecomment-3603827896), he as a distro maintainer is in favor of us making the build breaking change now.

I also lean towards doing this now than in a subsequent servicing release but I don't have strong feelings.

Reply:
Apologies @matouskozak, I didn't mean to mention you.  I intended to mention @mateusrodrigues.

Reply:
+1 on @omajid's comment and making this change now.